prisma/schema.prisma

```diff
@@ -58,6 +58,10 @@ model Application {
   paymentMethod                    PaymentType             @map("payment_method")
   processingFee                    Decimal                 @map("processing_fee") @db.Money
   donationAmount                   Decimal                 @default(0.00) @map("donation_amount") @db.Money
+  paymentMethod2                   PaymentType?            @map("payment_method2")
```

Comment:
Nit about the naming: what about `second_payment_method`, `second_processing_fee`, etc.? `paymentMethod2` almost makes it seem like we're using it in favour over a deprecated `paymentMethod` column.

Reply:
Good point! Renamed variables to improve clarity [here](https://github.com/uwblueprint/richmond-centre-for-disability/pull/338/commits/8e6fb0df2f4e1bee1f13ba14f237397797680861)